djangocms_versioning_filer/monkeypatch/admin/clipboardadmin.py

```diff
@@ -61,9 +66,28 @@ def ajax_upload(request, folder_id=None):
         if uploadform.is_valid():
             file_obj = uploadform.save(commit=False)
             # Enforce the FILER_IS_PUBLIC_DEFAULT
-            file_obj.is_public = filer_settings.FILER_IS_PUBLIC_DEFAULT
-            file_obj.folder = folder
+            file_obj.iNos_public = filer_settings.FILER_IS_PUBLIC_DEFAULT
 
+            # manage folder allocations
+            path = request.POST.get('path')
+            # import ipdb; ipdb.set_trace()
+            if path:
+                pathSplit = path.split('/')
+                # if no folders in supplied path, then add to current folder
+                if len(pathSplit) == 0:
```

Comment:
This will never happen - str.split always returns a list with at least one element
```
>>> "".split("/")
['']
>>> "aaa".split("/")
['aaa']
```